Jenkinsfile

```
@@ -73,7 +73,7 @@ String get_daos_packages() {
 String get_daos_packages(String distro) {
     String pkgs
     if (env.TEST_RPMS == 'true') {
-        pkgs = "daos{,-{client,tests,server}}"
+        pkgs = "daos{,-{client,tests,server,tests-java,devel,debuginfo}}"
```

We don't need the `devel` or `debuginfo` packages here.

```suggestion
        pkgs = "daos{,-{client,tests{,-java},server}}"
```

---

java maven run looks for daos headers and I think it might need daos-devel.

---

Does it **really** need the headers once everything is built?  Or is this a by-product of the questions about what `maven clean` is actually doing?

In any case, I would prefer adding the daos-devel to the daos-tests-java than adding them here, resulting in them being added to every node.

---

Not sure if it is related to clean but as you have mentioned now. It is quite possible. I will check it without clean once I have packaging stuff sorted out.

#In any case, I would prefer adding the daos-devel to the daos-tests-java than adding them here, resulting in them being added to every node.
So if I need to add them in daos-tests-java, it will go under %REQUIRES section?

---

> So if I need to add them in daos-tests-java, it will go under %REQUIRES section?

In a `Requires: daos-devel${?isa} = %{version}-%{release}` in the `daos-tests-java` `%package`.

But please confirm the need for that by removing `clean` first.